catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/common/CsexraAttributeTest.java

```diff
@@ -229,12 +229,840 @@ public void testSnowDepthMaxCategoryOther() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.SENSOR_NAME)).thenThrow(NitfFormatException.class);
+
+        Serializable actual = CsexraAttribute.SENSOR.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, nullValue());
+    }
+
+    @Test
+    public void testTimeFirstLineImageMin() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.TIME_FIRST_LINE_IMAGE_NAME)).thenReturn(
+                "00000.000000");
+
+        Serializable actual = CsexraAttribute.TIME_FIRST_LINE_IMAGE.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(Float.class)));
+        assertThat(actual, is(00000.000000f));
+
+    }
+
+    @Test
+    public void testTimeFirstLineImageMax() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.TIME_FIRST_LINE_IMAGE_NAME)).thenReturn(
+                "86400.000000");
+
+        Serializable actual = CsexraAttribute.TIME_FIRST_LINE_IMAGE.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(Float.class)));
+        assertThat(actual, is(86400.000000f));
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testTimeFirstLineNotSet() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.TIME_FIRST_LINE_IMAGE_NAME)).thenThrow(
+                NitfFormatException.class);
+
+        Serializable actual = CsexraAttribute.TIME_FIRST_LINE_IMAGE.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, nullValue());
+    }
+
+    @Test
+    public void testTimeImageDurationMin() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.TIME_IMAGE_DURATION_NAME)).thenReturn("-9999.999999");
+
+        Serializable actual = CsexraAttribute.TIME_IMAGE_DURATION.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(Float.class)));
+        assertThat(actual, is(-9999.999999f));
+
+    }
+
+    @Test
+    public void testTimeImageDurationMax() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.TIME_IMAGE_DURATION_NAME)).thenReturn("86400.000000");
+
+        Serializable actual = CsexraAttribute.TIME_IMAGE_DURATION.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(Float.class)));
+        assertThat(actual, is(86400.000000f));
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testTimeImageDurationNotSet() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.TIME_IMAGE_DURATION_NAME)).thenThrow(
+                NitfFormatException.class);
+
+        Serializable actual = CsexraAttribute.TIME_IMAGE_DURATION.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, nullValue());
+    }
+
+    @Test
+    public void testMaxGSDMin() throws NitfFormatException {
```

Comment:
And, Google code style guide also says thats the way you should do it. :)
https://google.github.io/styleguide/javaguide.html